src/docker-osx-dev

```diff
@@ -353,22 +353,27 @@ function check_for_shared_folders {
 
   if [[ ! -z "$vbox_shared_folders" ]]; then
     log_error "Found VirtualBox shared folders on your Boot2Docker VM. These may void any performance benefits from using docker-osx-dev:\n$vbox_shared_folders"
-    log_instructions "Would you like this script to remove them?"
-
-    local choice=""
-    select choice in "yes" "no"; do
-      case $REPLY in
-        y|Y|yes|Yes )
-          umount_vboxsf_mounted_folder "$vbox_shared_folders"
-          break
-          ;;
-        n|N|no|No )
-          log_instructions "Please remove the VirtualBox shares yourself and re-run this script. Exiting."
-          exit 1
-          ;;
-      esac
-    done
-  fi
+
+    if [[ "$remove_shared_folders" = true ]]; then
```

Comment:
@brikis98 - According to the [Advanced Bash-Scripting Guide]((http://tldp.org/LDP/abs/html/localvar.html#FTN.AEN18568)):
> a local variable declared in a function is also visible to functions called by the parent function.

I used this quirk to avoid passing the value down the function chain (`install/sync` -> `init_docker_host` -> `init_boot2docker/init_docker_machine` -> `check_for_shared_folders`) as that seemed excessive.

That being said- this may be more confusing than using global vars, so I can switch to passing it as an argument to each of the aforementioned functions, or back to a global variable.

An additional option would be to refactor the `init_boot2docker/init_docker_machine` and move the  `check_for_shared_folders` call into the `install/sync` functions.